packages/x-charts-pro/src/context/CartesianProviderPro/createAxisFilterMapper.ts

```diff
@@ -6,10 +6,10 @@ import {
   ZoomAxisFilters,
   GetZoomAxisFilters,
   isDefined,
+  getScale,
 } from '@mui/x-charts/internals';
 import { ChartsAxisProps, ScaleName, AxisConfig } from '@mui/x-charts';
 import { ZoomData } from '../ZoomProvider';
-import { getScale } from '../../../../x-charts/src/internals/getScale';
```

Comment:
Didn't eslint pick this up? 😮

Reply:
Somehow it didn't. 🤔 
Could be because of the issues in the ESLint config that are pending resolution: https://github.com/mui/mui-x/pull/13401